wen-restart/src/epoch_stakes_cache.rs

```diff
@@ -0,0 +1,129 @@
+use {
+    solana_program::{clock::Epoch, epoch_schedule::EpochSchedule},
+    solana_runtime::{bank::Bank, epoch_stakes::EpochStakes},
+    solana_sdk::{clock::Slot, pubkey::Pubkey},
+    std::{collections::HashMap, sync::Arc},
+};
+
+pub struct EpochStakesCache {
+    epoch_stakes_map: HashMap<Epoch, EpochStakes>,
+    epoch_schedule: EpochSchedule,
+}
```

Comment:
I think this functionality if we're gonna build this module should:

1. live in solana runtime 
2. just replace the `epoch_stakes: HashMap<Epoch, EpochStakes>` in the bank

Reply:
This class used to do much more, but now that we only need epoch_schedule and epoch_stakes of root_bank, and we don't need to switch to another bank, I think it makes sense to just remove this class.

It might make sense to add node_id_to_stake to EpochStakes, let me figure out how to create EpochStakes in tests, it's so tangled with other classes.

Reply:
Added node_id_to_stake to EpochStakes, does it look better now?